vendor/github.com/denisbrodbeck/machineid/id_unsupported.go

```diff
@@ -0,0 +1,12 @@
+// +build !freebsd,!netbsd,!openbsd,!dragonfly,!windows,!linux,!darwin
```

Comment:
I don't think we can directly patch vendor code without it leading to problems later. There's a bit about our module/vendoring here: https://github.com/buildkite/agent#dependency-management

A pull request to https://github.com/denisbrodbeck/machineid might be a better option.

Reply:
Ye, I thought about that first but seeing the how stale the upstream is, I took a shot to have it patched here. I will submit this patch to upstream as well.

Reply:
Actually, came to think of it — the right approach is probably to abstract away the machineid lib and use build tags in this project.